File: BACKEND/BE/ArchivoAdjunto.cs
using System;
using System.Collections.Generic;

namespace BE;

public partial class ArchivoAdjunto
{
    public int Id { get; set; }

    public DateTime FechaSubida { get; set; }

    public string NombreArchivo { get; set; } = null!;

    public int? Tamano { get; set; }

    public string? Url { get; set; }

    public int EstudioId { get; set; }

    public bool Activo { get; set; }

    public virtual Estudio Estudio { get; set; } = null!;
}

File: FRONTEND/src/app/features/paciente/interfaces/archivoAdjunto.ts
export interface ArchivoAdjunto {
    id: number,
    fechaSubida?: Date | null,
    nombreArchivo: string,
    tamano?: number | null,
    url?: string | null,
    estudioId: number,
    estudioTipoNombre?: string | null,
    activo: number
}
